CMakeLists.txt

```diff
@@ -527,9 +543,9 @@ target_include_directories(onnx PUBLIC
 target_link_libraries(onnx PUBLIC onnx_proto)
 add_onnx_global_defines(onnx)
 
-if(BUILD_ONNX_PYTHON)
-  if("${PY_EXT_SUFFIX}" STREQUAL "")
-    if(MSVC)
+if(ONNX_BUILD_PYTHON)
+  if(NOT PY_EXT_SUFFIX)
+    if(WIN32)
```

Comment:
Could you share the reason for changing msvc to win32? Thanks - just for my education